spec/factories/clients.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :client do
+    email { Faker::Internet.email }
```

Comment:
@DmitrS it's better to use `safe_email` to be sure that test email won't be accidentally delivered to a random person.